atomix/cluster/src/main/java/io/atomix/raft/storage/log/RaftLogWriter.java

```diff
@@ -17,13 +16,6 @@
 package io.atomix.raft.storage.log;
 
 import io.atomix.raft.storage.log.entry.RaftLogEntry;
-import io.atomix.storage.journal.DelegatingJournalWriter;
-import io.atomix.storage.journal.SegmentedJournalWriter;
+import io.atomix.storage.journal.JournalWriter;
 
-/** Raft log writer. */
-public class RaftLogWriter extends DelegatingJournalWriter<RaftLogEntry> {
-
-  public RaftLogWriter(final SegmentedJournalWriter<RaftLogEntry> writer) {
-    super(writer);
-  }
-}
+public interface RaftLogWriter extends JournalWriter<RaftLogEntry> {}
```

Comment:
This interface is not used anymore.

Reply:
:see_no_evil: That was the whole point of the PR! Sorry haha